### ISSUE TITLE
Request for additional commands

### ISSUE THREAD

**Issue by BanterBoy** [trossr32/ps-transmission#10]
Couple of extra commands to consider adding to this module.

```powershell
Enable-TransmissionSpeedLimits
Disable-TransmissionSpeedLimits
````
It would be useful to be able to disable/enable the speedlimits from the command line

**Comment by trossr32** [comment]
Thanks for raising this, as it highlights a missing endpoint that has not been included which is to set session config.

Docs here for reference: https://github.com/transmission/transmission/blob/2.9x/extras/rpc-spec.txt#L532

Tasks required to resolve issue:

- [x] Add Set-TransmissionSession cmdlet
- [x] Test working features in current available version (2.94 for me)

@BanterBoy In order to add the bespoke commands requested please advise whether you would expect the commands to enable one or both of the following:

- download speed limit
- upload speed limit

**Comment by trossr32** [comment]
New cmdlets added:

Set-TransmissionSession
Set-TransmissionAltSpeedLimits

Released as v1.0.8
